yktrace/src/sir.rs

```diff
@@ -9,9 +8,11 @@ use std::{
     fmt::{self, Debug, Display, Write},
     io::Cursor,
     iter::Iterator,
-    path::Path
+    path::Path, fs::File
 };
 use ykpack::{self, bodyflags, Body, CguHash, Decoder, Local, Pack, Ty};
+use memmap::Mmap;
+use object::{ObjectSection, Object}; // FIXME kill.
```

Comment:
What's the FIXME about?

Reply:
Huh, I can't remember writing that.

Reply:
What is even weirder is that that line was created using the rust-analyzer import assist as far as I know.